tests/testthat/test-model-options.R

```diff
@@ -64,12 +64,17 @@ test_that("do_endpoint_model_options correctly builds params list", {
                  "calendar_quarter_t1_options",
                  "calendar_quarter_t1_default",
                  "calendar_quarter_t2_options",
-                 "survey_prevalence_options", "survey_art_coverage_options",
+                 "survey_prevalence_options", "survey_prevalence_default",
+                 "survey_art_coverage_options", "survey_art_coverage_default",
```

Comment:
Putting these on same line is discrepant with similar cases before and after. Suggest one each line.